crates/ruff/src/rules/flake8_self/rules/private_member_access.rs

```diff
@@ -9,6 +9,42 @@ use crate::registry::Diagnostic;
 use crate::violation::Violation;
 
 define_violation!(
+    /// ## What it does
+    /// Checks for the access of a private member of a class.
+    ///
+    /// ## Why is this bad?
+    /// If a member of a class is declared private, the standard is that
+    /// the member in question generally shouldn't be accessed by anything
+    /// except for internally in the class. Using private variables can
+    /// also possibly cause problems if used incorrectly, and those errors
+    /// can be difficult to debug.
+    ///
+    /// Instead, the name of the member should be renamed to public
+    /// (no leading underscores) if possible. If this is not possible,
+    /// consider removing the usage or adding a `noqa` statement.
```

Comment:
What to do depends on a case by case basis. We should not recommend making the member public as the primary solution.